src/licence-status-banner/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import Snippet from '../snippet';
+import classnames from 'classnames';
+import formatDate from 'date-fns/format';
+
+class LicenceStatusBanner extends Component {
+  componentDidMount() {
+    this.setState({ open: false });
+  }
+
+  toggle() {
+    return this.setState({ open: !this.state.open });
+  }
+
+  isOpen() {
+    return !this.state || this.state.open;
+  }
+
+  render() {
+    const licence = this.props.licence;
+    const licenceType = this.props.licenceType;
+    const dateFormat = this.props.dateFormat;
+
+    if (licence.status === 'active') {
+      return null;
+    }
+
+    return (
+      <div className={classnames('licence-status-banner', licence.status, this.isOpen() ? 'open' : '')}>
```

Comment:
`classnames` can take literally anything you throw at it - you could write this as:
```js
classnames('licence-status-banner', licence.status, {  open: this.isOpen() })
```

Reply:
cool, didn't know that, ta.

Reply:
SHIP IT